R/util.R

```diff
@@ -1,6 +1,12 @@
 pkg.env = new.env()
 
-pkg.env$baseURL = "https://cellengine.com/api/v1"
+# Set your development base url in .Renviron in the package directory
+dev.env = Sys.getenv('CELLENGINE_DEVELOPMENT')
```

Comment:
Can you call this `CELLENGINE_API_URL` maybe, to be explicit?

You can also use `Sys.getenv("varname", "https://cellengine.com/api/v1")` (i.e. make use of the poorly named `unset` parameter) instead of the if/else below.